src/main/java/com/google/sps/servlets/AccountData.java

```diff
@@ -23,11 +23,20 @@
     private final boolean loggedIn;
     private final String logInOutURL;
     private final String signUpURL;
+    private final UserInfo userInfo;
 
     public JsonAccountInfo(Boolean loggedIn, String logInOutURL, String signUpURL) {
```

Comment:
See Effective Java 3, Item 1: Consider static factory methods instead of constructors

Use 2 factory methods with clear names.

Reply:
What would be the advantage/disadvantage of using each?

Reply:
nevermind, i looked at the book you referenced and it lists all the advantages/disadvantages there